server/app.js

```diff
@@ -3,7 +3,7 @@ const cors = require('cors');
 const db = require('./db/index.js');
 require('dotenv').config();
 
-const port = process.env.PORT;
+const port = 3004;
```

Comment:
Is there a reason you removed your environment variable here?

Reply:
I had some issues and will refactor with it.